### PR TITLE
Minor tweaks to workloads; increase data sizes.

### PR DESCRIPTION
**Pull request by pbailis**
@deepakn94 can you check to see if you agree with these modifications?

### CODE REVIEW

File: bench/conf/workflow_config.json [hunk 1]
```diff
@@ -53,7 +51,19 @@
   },
   {
     "targetLowMetrics": [],
-    "baseQuery": "SELECT * FROM uk_road_accidents LIMIT 1000000;",
+    "baseQuery": "SELECT * FROM uk_road_accidents LIMIT 10000000;",
+    "isBatchJob": true,
+    "targetAttributes": [
+      "road_surface_conditions"
+    ],
+    "taskName": "accidentsSimple",
```
Comment (deepakn94) [inline bench/conf/workflow_config.json]: Two workloads with the same name...

Comment (pbailis) [inline bench/conf/workflow_config.json]: Fixed, thanks.

**Comment by deepakn94** [comment]
LGTM.

**Comment by pbailis** [comment]
02b49af7eb9529f4053382d55569a2130e8d6b04

**Pull request closed without merging**